web_infrastructure/jboss.py

```diff
@@ -21,7 +21,7 @@
 DOCUMENTATION = """
 module: jboss
 version_added: "1.4"
-short_description: deploy applications to JBoss
+short_description: deploy applications to JBoss 7 EAP
```

Comment:
This also works for EAP 6, so I think it would be best to clarify that it works for older versions than version X (where X should probably be both the Wildfly version and the corresponding EAP version)